scan/src/images/Images.re

```diff
@@ -37,3 +37,4 @@
 [@bs.module] external leftAngle: string = "./left-angle.svg";
 [@bs.module] external graphBG: string = "./graph-bg.svg";
 [@bs.module] external notFoundBg: string = "./notFoundBg.svg";
+[@bs.module] external txLogo: string = "./tx-icon.png";
```

Comment:
Already declare?

Reply:
it's a transparent background, maybe I will change the name for not miss communication?